### PR TITLE
Replace gaffe with giraffe

### PR DESCRIPTION
**Pull request by xchang1**
rename gaffe -> giraffe

### CODE REVIEW

**Comment by adamnovak** [comment]
I've restarted Mac Travis for this. They both ran out of time during the tests. It could be just cloud weather, or it could be that something here introduces a hang in the Mac tests.

File: scripts/giraffe-facts.py [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 
 """
-giraffe-facts.py: process a GAM file from the new minimizer-based mapper (vg
-gaffe AKA giraffe) and report runtime statistics by filter.
+giraffe-facts.py: process a GAM file from the new minimizer-based mapper (vggiraffe) and report runtime statistics by filter.
```
Comment (adamnovak) [inline scripts/giraffe-facts.py:6]: This looks to be missing a space.

**Comment by glennhickey** [comment]
I'm seeing Mac Travis hang in another PR here:
https://github.com/vgteam/vg/pull/2813

**Comment by adamnovak** [comment]
I think it was cloud weather. The rerun Mac test tool ~46 minutes instead of an hour and a half.

**Comment by adamnovak** [comment]
@xchang1 Do you want to make a corresponding toil-vg fix to make it call `vg giraffe`? Or should we include a backward-compatibility alias?

**Comment by xchang1** [comment]
I can change it in toil-vg

**Pull request merged**